### PR TITLE
Deprecations and removals for Chrome 106.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by jpmedley** [comment]
@devnook Can you please look at this. It's failing because it can't find a result for i18n.tags.removals. We updated some tags in #3383. This must have an old version cached.

**Comment by jpmedley** [comment]
@devnook Or can you assign it to someone on your team? This needs to go out in about 27 hours.

File: site/en/blog/deps-rems-106/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tags:
+  - deprecations-removals
+  - chrome-106
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 106 beta was released on September 1, 2022 and is expected to become the stable version in late September, 2022.
+
+## Remove non-ASCII characters in cookie domain attributes {#remove-non-ascii-characters-in-cookie-domain-attributes}
```
Comment (tunetheweb) [inline site/en/blog/deps-rems-106/index.md:19]: Anchors for headings need to be `{: #anchor }` and you missed the colon and space which was causing build to fail.

But honestly you don't need them - it'll auto generate them for you. I fixed this up recently where it wasn't stripping some punctuation characters out but should be all good now. So let's remove.

```suggestion
## Remove non-ASCII characters in cookie domain attributes
```

File: site/en/blog/deps-rems-106/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+To align with the latest spec ([RFC 6265bis](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis/#section-5.5)), [Chromium now rejects](https://www.chromestatus.com/feature/5534966262792192) cookies with a `Domain` attribute that contains non-ASCII characters (for example, `éxample.com`).
+
+Support for IDN domain attributes in cookies has been long unspecified, with Chromium, Safari, and Firefox all behaving differently. This change standardizes Firefox's behavior of rejecting cookies with non-ASCII domain attributes.
+
+Since Chromium has previously accepted non-ASCII characters and tried to convert them to normalized punycode for storage, we will now apply stricter rules and require valid ASCII (punycode if applicable) domain attributes.
+
+## Remove HTTP/2 push {#remove-http-2-push}
```
Comment (tunetheweb) [inline site/en/blog/deps-rems-106/index.md:27]:
```suggestion
## Remove HTTP/2 push
```

File: site/en/blog/deps-rems-106/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Chrome has [removed the ability](https://www.chromestatus.com/feature/6302414934114304) to receive, keep in memory, and use HTTP/2 push streams sent by the server. See [Removing HTTP/2 Server Push from Chrome](/blog/removing-push/) for details and suggested alternative APIs.
+
+## Remove Persistent Quota  {#remove-persistent-quota}
```
Comment (tunetheweb) [inline site/en/blog/deps-rems-106/index.md:31]:
```suggestion
## Remove Persistent Quota
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: site/en/blog/deps-rems-106/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/blog/deps-rems-106/index.md:19]: I used a different markdown converter. That's what it did and it didn't occur to me to look at them closesly.

**Pull request closed without merging**